nerfstudio/viewer/app/src/modules/ViewerWindow/ViewerWindow.jsx

```diff
@@ -191,7 +191,7 @@ export default function ViewerWindow(props) {
         const oldBackground = scene.background;
         const texture = new THREE.Texture(this);
         texture.minFilter = THREE.LinearFilter;
-        texture.magFilter = THREE.NearestFilter;
+        texture.magFilter = THREE.LinearFilter;
```

Comment:
don't feel strongly but would be cleaner to move changes that are orthogonal to viewercontrols to their own PRs